### DISCUSSION TITLE
What is the default coordinate system?

### DISCUSSION THREAD

**Discussion by carlbean** [stella-cv/stella_vslam#157] [Q&A] [answered]
I assume it's right-handed y-down z-forward but I can't find any evidence :)

**Comment by ymd-stella** [comment, marked as answer]
> I assume it's right-handed y-down z-forward

Yes, OpenVSLAM uses it. It is the same as the OpenCV coordinate system.

**Comment by carlbean** [comment]
Thank you!!
